### ISSUE TITLE
Add mplcal to pypi

### ISSUE THREAD

**Issue by meta4** [meta4/mplcal#6]
A few arguments for and against.

- Pro: It would be easier for others to find and use mplcal.
- Pro: It would be cool to have a package on pypi, mplcal would be my first.
- Con: mplcal was created just to [scratch an itch](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar#Lessons_for_creating_good_open_source_software).
- Con: It probably has bugs; [theWasweisich](https://github.com/meta4/mplcal/commits?author=theWasweisich) just found a rather [obvious bug](https://github.com/meta4/mplcal/commit/8706b4ae68cb1243dc48cd740c9deff1a36a23ca) in the error handling (which is cool, but humbling).
- Con: It requires matplotlib to work.  For someone who wants to make a calendar, this is not an obvious requirement.
- Con: I don't know how to add a package to pypi, or maintain & update a pypi package.

This issue is to get a sense of how helpful a pypi package would be.

If you think it would be helpful to add mplcal to pypi please add a "me too" comment on this issue.

If you have other pro or con arguments add them to the issue.

**Comment by SurajBhari** [comment]
me too

**Comment by SurajBhari** [comment]
> It requires matplotlib to work. For someone who wants to make a calendar, this is not an obvious requirement.

I don't agree with this point there are so many dependencies nowadays that it barely matters. python is slowly turning into what happened with any JS framework. hundreds and hundreds of dependencies. outdated depricated. 

I get the code quality point of view. but ease of access also make it better. 

at least in this case it's a well-known library. I don't think anyone is hesitant to use a well-known library. in many case its already present in the system.
